lib/svg-sprite.js

```diff
@@ -249,40 +248,33 @@ SVGSpriter.prototype._compile = function () {
 
             that._logStats(files);
             args[1](null, files, {});
-
-            // Else
         } else {
-            var masterShapes = this._shapes.filter(function (shape) {
-                return !shape.master;
-            }).length;
-            this.info('Compiling %d shapes ...', masterShapes);
+            const masterShapes = this._shapes.filter(shape => !shape.master).length;
+            this.info('Compiling %d shapes...', masterShapes);
 
             // Initialize the namespace powers
-            while (!this._namespacePow.length || (Math.pow(26, this._namespacePow.length) < masterShapes)) {
-                this._namespacePow.unshift(Math.pow(26, this._namespacePow.length));
+            while (!this._namespacePow.length || (26 ** this._namespacePow.length < masterShapes)) {
+                this._namespacePow.unshift(26 ** this._namespacePow.length);
                 _.invoke(this._shapes, 'resetNamespace');
             }
 
             // Sort shapes by ID
             this._shapes = this._shapes.sort(this.config.shape.sort);
 
             // Set the shape namespaces on all master shapes
-            this._shapes.filter(function (shape) {
-                return !shape.master;
-            }).forEach(function (shape, index) {
-                shape.setNamespace(this._indexNamespace(index));
-            }, this);
-
-            this._layout(args[0], function (error, files, data) {
+            this._shapes.filter(shape => !shape.master)
+                .forEach(function(shape, index) {
+                    shape.setNamespace(this._indexNamespace(index));
+                }, this);
 
+            this._layout(args[0], (error, files, data) => {
                 // Add intermediate SVG files
                 if (that.config.shape.dest) {
                     files.shapes = that._getShapeFiles(that.config.shape.dest);
                     that.verbose('Returning %d intermediate SVG files', files.shapes.length);
                 }
-                that.info('Finished %s sprite compilation', Object.keys(data).map(function (mode) {
-                    return '«' + mode + '»';
-                }).join(' + '));
+
+                that.info('Finished %s sprite compilation', Object.keys(data).map(mode => '«' + mode + '»').join(' + '));
```

Comment:
We could probably replace `'«' + mode + '»'` with a template string, right? Minor thing though …

Reply:
Yup, I played with the idea of enforcing template literals, but they barely looks more readable in only a few cases. So, I decided to leave it out for the time being.

Reply:
See https://github.com/svg-sprite/svg-sprite/commit/2f66c4fa529d49e1f70d73f8b591ce2ed494e76d

I can pick this patch in this branch later if you want, otherwise I'll just land it separately.

Reply:
Whatever works best for you!